server/services/connection-service.js

```diff
@@ -58,14 +59,13 @@ module.exports = {
     }
   },
   isConnected: async function () {
-    if (connections.length !== 2) {
+    if (connections.length !== EXPECTED_CONNECTIONS) {
+      return false
+    }
+    const inactiveConnections = connections.find(x => !x.isOpen())
+    if (inactiveConnections !== undefined) {
       return false
     }

```

Comment:
you could achieve the same logic with:

`return connections.some(x=> !x.isOpen())`

Also the async can be removed from the function signature

Reply:
Removed async.

Didn't know about "some" so thanks for that.  But wouldn't the above return the wrong value?

If any connection is not open this will return true, but we'd actually want it to return false.

`return !connections.some(x=> !x.isOpen())` would work though.